src/main/io/ledstrip.c

```diff
@@ -484,20 +478,32 @@ static void applyLedFixedLayers()
         }
 
         if (ledGetOverlayBit(ledConfig, LED_OVERLAY_THROTTLE)) {
-            hOffset += scaledAux;
+			if (rcData[THROTTLE] < (PWM_RANGE_MIN + PWM_RANGE_MAX)/2) //if below average
```

Comment:
https://www.youtube.com/watch?v=RtY9wzNQD9A this shows the effect I achieved, using 4 x 3 fades with the same starting point but different mid point and top end